app/views/simplified-account/settings/email-notifications/index.njk

```diff
@@ -39,11 +39,12 @@
       },
       {
         key: {
-        text: "Payment confirmation emails"
-      },
+          text: "Payment confirmation emails"
+        },
         value: {
-        text: 'On' if confirmationEmailEnabled else 'Off'
-      },
+          text: 'Yes' if (confirmationEmailEnabled and emailCollectionMode === 'MANDATORY') else
+            ('No' if (not confirmationEmailEnabled and emailCollectionMode === 'MANDATORY') else 'No - email notification off')
```

Comment:
Just realised this logic isn't quite right, it should be `and emailCollectionMode !== 'OFF'`, so it doesn't show `No - email notification off` when email collection is optional

Reply:
Yeah realised this when doing an [upcoming PR](https://github.com/alphagov/pay-selfservice/pull/4315/files#diff-27c9ea0cf0d16cd3989d0d7e6f96cc316d1d06b2cf361888959f165b92d12156R45-R47). I'll just save that for that cypress test PR?

Reply:
Nevermind, I've changed this PR